etc/kayobe/stackhpc-monitoring.yml

```diff
@@ -53,3 +53,18 @@ redfish_exporter_default_password: "{{ ipmi_password }}"
 redfish_exporter_target_address: "{{ ipmi_address }}"
 
 ###############################################################################
+
+# Whether the RADOS gateway usage exporter is enabled.
+# Enabling this will result in templating radosgw_usage_exporter endpoint as
+# Prometheus scrape targets during deployment.
+stackhpc_enable_radosgw_usage_exporter: false
+
+# Port to expose RADOS gateway usage exporter. Default is 9242
+stackhpc_radosgw_usage_exporter_port: 9242
+
+# Path to a certificate for internal TLS in the RADOS gateway usage exporter.
+stackhpc_radosgw_usage_exporter_cacert: ""
+
+# Whether TLS certificate verification is enabled for the RADOS gateway usage
+# exporter for querying Ceph RADOS gateway APIs. Default is true
+stackhpc_radosgw_usage_exporter_verify: true
```

Comment:
Seems strange that the default is true but there's no default cert or cert path.
Would it make sense for it to be dependent on something like `kolla_enable_tls_internal`?